react/components/FormHandler.tsx

```diff
@@ -35,6 +36,11 @@ export const FormHandler: FC<{
       }
       dispatchSubmitAction({ type: 'SET_LOADING' })
 
+      data = parseDateTimeFieldsData({
```

Comment:
It's not a really good practice to reassign parameters. I think you should create a new const for this.

Reply:
You're right, fixed. Thank you for the review 😃